CONTRIBUTING.md

```diff
@@ -98,7 +98,7 @@ git checkout devel
 git pull
 ```
 
-Now you can branch from `devel` to create new features and when you push those feature branches they will push to your fork. When you pull devel, it will get updates from the official sst repo
+Now you can branch from `devel` to create new features and when you push those feature branches they will push to your fork. When you pull devel, it will get updates from the official sst repo, avoiding the need to merge changes in devel into your local fork
```

Comment:
Period at end of the sentence and maybe "avoiding the need to merge changes from the main repository into your local fork."